### PR TITLE
fix #615

### LINKED ISSUE #615: Use PartitionId in TaskStatus
See

https://github.com/ballista-compute/ballista/blob/a0c609f3a43ed316df6c21c3e04ce7de726455d4/rust/ballista/proto/ballista.proto#L486-L496

### PR DESCRIPTION
**Pull request by vigsterkr**
use PartitionId in TaskStatus

### CODE REVIEW

File: rust/ballista/src/bin/executor.rs [hunk 1]
```diff
@@ -77,9 +77,11 @@ async fn poll_loop(
...
                     *running_task_guard = None;
                     task_status.push(TaskStatus {
-                        job_id: task_id.job_id,
-                        stage_id: task_id.stage_id,
-                        partition_id: task_id.partition_id,
+                        partition_id: Some(PartitionId {
+                            job_id: task_id.job_id,
+                            stage_id: task_id.stage_id,
+                            partition_id: task_id.partition_id,
+                        }),
```
Comment (edrevo) [inline rust/ballista/src/bin/executor.rs:84]: can't you do this? (maybe you'll need to clone)

```suggestion
                        partition_id: Some(task_id),
```

File: rust/ballista/src/bin/executor.rs [hunk 2]
```diff
@@ -90,9 +92,11 @@ async fn poll_loop(
...
```
Comment (edrevo) [inline rust/ballista/src/bin/executor.rs:99]: same comment, maybe you can just re-use task_id?

File: rust/ballista/src/bin/executor.rs (hunk 1 above)
Comment (vigsterkr) [reply rust/ballista/src/bin/executor.rs]: 🤦 definitely

**Review by edrevo — APPROVED** [review]

**Comment by edrevo** [comment]
Many thanks @vigsterkr

**Pull request merged**